src/main/resources/changesets/changelog-consent-36.0.xml

```diff
@@ -5,11 +5,12 @@
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet author="grushton" id="114">
+        <validCheckSum>ANY</validCheckSum>
         <addColumn tableName="consents">
-            <column name="dataUse" type="text"/>
+            <column name="datause" type="text"/>
         </addColumn>
-        <sql dbms="hsql, mysql">UPDATE consents SET dataUse = ''</sql>
-        <sql dbms="hsql, mysql">ALTER TABLE consents MODIFY dataUse text NOT NULL</sql>
+        <sql dbms="mysql">UPDATE consents SET datause = ''</sql>
+        <sql dbms="mysql">ALTER TABLE consents MODIFY datause text NOT NULL</sql>
```

Comment:
other places where you had dbms="hsql, mysql" you remove both... do you need mysql here?

Reply:
This could have used some more explanation in the PR description. In all of the cases where we changed the `dbms` field, these are my reasons. 

#### `dbms="mysql"` cases
These are the cases where I wanted to preserve some history since we made real changes to the mysql database. We need that field to restrict the sql so it will only run when in a mysql environment. We don't want these to run in either hsqldb or postgres for two reasons. When populating a new db (hsqldb, posgres), that data doesn't exist and never did. It was only ever used to migrate mysql data. There's no harm in running it for hsqldb, it works, but in a lot of cases, it fails in postgres due to syntax differences. In this case, the alter table fails. 

#### `dbms="hsqldb, mysql"` cases
In these cases, my intention was also to preserve some history. To be honest, some of the following ones could just be restricted to mysql as well. They either fail in postgres as they are written or are not necessary. For example, this fails in postgres (see next comment): `update vote set type = 'DAC' where isFinalAccessVote = 0;`. And like the above reason, we need to exclude these changes from postgres, so we need to specify which environments they should be run in.